src/reducers/errors.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import reducer, { actions, initialState } from './errors';
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('stores an error to show', () => {
```

Comment:
Further to my naming comment below, the test says "stores an error", but the action is called `showError`.

Reply:
hehe yes... so add/remove?